app/src/main/java/com/mopub/nativeads/InneractiveAdapterConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package com.mopub.nativeads;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import com.fyber.inneractive.sdk.external.InneractiveAdManager;
+import com.mopub.common.BaseAdapterConfiguration;
+import com.mopub.common.OnNetworkInitializationFinishedListener;
+import com.mopub.common.Preconditions;
+import com.mopub.common.logging.MoPubLog;
+import com.mopub.mobileads.MoPubErrorCode;
+
+import java.util.Map;
+
+import static com.fyber.inneractive.sdk.external.InneractiveMediationDefs.REMOTE_KEY_APP_ID;
+import static com.mopub.common.logging.MoPubLog.AdapterLogEvent.CUSTOM_WITH_THROWABLE;
+
+public class InneractiveAdapterConfiguration extends BaseAdapterConfiguration {
+
+  private static final String MOPUB_NETWORK_NAME = "inneractive";
```

Comment:
Did they provide you this adapter somewhere? I could not find it. The name of the network in MoPub is Fyber. I don't know if this will afffect or not, but whenever I implemented one non-certified network I always use the same name that they use to configure on mopub.

Reply:
Yeah, if you download the SDK, there's a MoPub folder (or something like that) that has all of these files.